src/android/RouteListComparer.java

```diff
@@ -0,0 +1,11 @@
+package acidhax.cordova.chromecast;
+
+import androidx.mediarouter.media.MediaRouter;
+
+import java.util.Comparator;
+
+public class RouteListComparer implements Comparator<MediaRouter.RouteInfo> {
+    public int compare(MediaRouter.RouteInfo left, MediaRouter.RouteInfo right) {
+        return left.getName().compareTo(right.getName());
```

Comment:
It might be a good idea to add checks for if `left` or `right` are `null` to avoid potential `NullPointerException`s here.

Reply:
I think Collections.sort does not call compare when there aren't enough items to compare with. So if the list is empty or there isn't two items left to compare, the function isn't evaluated.

Where null checks would be important is on getName(), but according to the API [1], it does not return null if empty like it does for getDescription() [2] or getExtras() [3] for example. In the unexpected event of an empty name, it most likely will by an empty String which the compareTo function will still accept.

[1] https://developer.android.com/reference/android/support/v7/media/MediaRouter.RouteInfo.html#getname
[2] https://developer.android.com/reference/android/support/v7/media/MediaRouter.RouteInfo.html#getdescription
[3] https://developer.android.com/reference/android/support/v7/media/MediaRouter.RouteInfo.html#getextras